src/Sylius/Bundle/ApiBundle/CommandHandler/PickupCartHandler.php

```diff
@@ -111,4 +119,16 @@ private function provideCustomer(): ?CustomerInterface
 
         return null;
     }
+
+    private function hasLocaleWithLocaleCode(ChannelInterface $channel, ?string $localeCode)
+    {
+        $locales = $channel->getLocales();
+
+        return !$locales
+            ->filter(function (LocaleInterface $locale) use ($localeCode): bool {
+                return $locale->getCode() === $localeCode;
+            })
+            ->isEmpty()
+        ;
```

Comment:
My recommendation, to improve readibility:

```suggestion
        $localeWithExcpectedCode = $locales->filter(function (LocaleInterface $locale) use ($localeCode): bool {
             return $locale->getCode() === $localeCode;
        });
        
        return !$localeWithExcpectedCode->isEmpty();
```